R/redcap_dates.R

```diff
@@ -0,0 +1,80 @@
+#' REDCap Date Conversion
+#'
+#' This function is trying to take all (most)
```

Comment:
i wonder if you should be more explicit that this is to do with data prep for import... as oposed to rc_date which is intended to be used on exported data...

Reply:
done